modules/apps/journal/journal-web/src/main/java/com/liferay/journal/web/internal/info/item/provider/JournalArticleInfoItemLanguagesProvider.java

```diff
@@ -31,6 +39,19 @@
 		return journalArticle.getAvailableLanguageIds();
 	}
 
+	@Override
+	public Set<Locale> getAvailableLanguages(
```

Comment:
This is wrong. There's no need to create a new method; the implementation is virtually the same as the one for the already existing method. Also, journal articles have nothing to do with layouts.